Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/VAVSingleZoneTSupSet.mo

```diff
@@ -500,6 +498,16 @@ economizer dampers.
 Note that the inputs <code>uHea</code> and <code>uCoo</code> must be computed
 based on the same temperature sensors and control loops
 </p>
+<p>
+fixme, mg notes: 
```

Comment:
Address and remove `fixme`. Once the code is merged, they otherwise won't get address easily and can hold up code releases.